src/Symfony/Bridge/Twig/Resources/views/Email/default

```diff
@@ -0,0 +1 @@
+zurb_2
```

Comment:
why is this file ?

Reply:
if this is a symlink, please test compat with Windows. Symlinks may not work there (using them in recipes is fine, because the source repo for recipes is not cloned by users but by the Flex server, so we don't care about cross-platform support there).

Reply:
This is indeed a symlink. I know that this will probably not work on Windows, not sure how to do otherwise.

Reply:
well, instead of having a `default` theme being a symlink to zurb_2, why not having a (private) constant in the NotificationEmail to define the name of the default theme (and then loading templates with `zurb_2` in the name) ?

Reply:
Because I want people to be able to override the `default` theme by creating `templates/emails/default/...` files.

Reply:
well, then I don't see a solution to support Windows if that's the requirement...

Reply:
Why we keep the `zurb_2` directory? could it be named `default` instead?

Reply:
fixed now.